rs/nervous_system/integration_tests/tests/sns_upgrade_test_utils_legacy.rs

```diff
@@ -98,19 +125,6 @@ pub async fn test_sns_upgrade_legacy(sns_canisters_to_upgrade: Vec<SnsCanisterTy
         .await;
     }
 
-    eprintln!("Await the swap lifecycle ...");
```

Comment:
Note to reviewer. 

It's not possible to disable automatic target version advancement in pre-initialization mode, so to get as close as possible to the old behavior, we need to first finalize the swap, then disable automatic target version advancement, then add the Wasms to SNS-W, then run the code under test. 

Previously, we first added the Wasms to SNS-W, then finalized the swap, and then ran the code under test.

Reply:
It would have helped if you had said online 47 that you just changed the ORDER of operations (and ofc, why). Otherwise, it looks like new code, and it is not immediately clear why new code is needed. The truth is that the code is NOT new; just transplanted from within the same test.